kratos/utilities/element_size_calculator.cpp

```diff
@@ -116,6 +116,105 @@ double ElementSizeCalculator<2,3>::MinimumElementSizeDerivative(
     KRATOS_CATCH("");
 }
 
+// Triangle2D6 version.
+template<>
+double ElementSizeCalculator<2,6>::MinimumElementSize(const Geometry<Node<3> >& rGeometry)
```

Comment:
As far as I see the implementation of the quadratic triangle is exactly the same as that of the linear one. If so it is, I'd call the other method in here.